src/main.rs

```diff
@@ -50,9 +54,11 @@ mod version;
 mod vwmap;
 
 fn main() {
+    Builder::new().filter_level(LevelFilter::max()).init();
```

Comment:
another issue here is that the level is set to TRACE by default, which is undesired for production... The default should be INFO while allowing the user to change the log level if they choose to change an env variable. So I would do something like this:

`
fn main() {
let mut builder = Builder::new();
let log_level = std::env::var("LOG_LEVEL").unwrap_or_else(|_| "info".to_string());
match log_level.to_lowercase().as_str() {
"info" => builder.filter_level(LevelFilter::Info),
"warn" => builder.filter_level(LevelFilter::Warn),
...
_ => builder.filter_level(LevelFilter::Info), 
};
builder.init();
`
(did not test this code obviously)

Reply:
Thanks, makes sense yes -- modified your snipped a bit + abstracted it away to `logging_layer.rs` due to main.rs/lib.rs both consuming the same thing (which we want to guarantee)